catalog/pipeline/release/0.8/README.md

```diff
@@ -0,0 +1,76 @@
+# Release Pipeline
+
+Tekton pipeline to release HACBS Snapshot to Quay.
```

Comment:
Maybe this is a good time to change HACBS to Stonesoup?

Reply:
fixed